examples/palm/python/permissions/sharing.py

```diff
@@ -0,0 +1,26 @@
+"""Sample functions illustrating how to share models using the permission service."""
+import google.generativeai as palm
+import google.ai.generativelanguage as glm
+from google.ai.generativelanguage_v1beta3 import types
```

Comment:
Is this still correct with the latest SDK updates?

Reply:
This shouldn't be necessary. the current default (`v1beta`) is a superset of this old one so `glm` is all you need.